sky/packages/sky/lib/gestures/arena.dart

```diff
@@ -32,6 +32,8 @@ class GestureArenaEntry {
 class GestureArena {
   final Map<Object, List<GestureArenaMember>> _arenas = new Map<Object, List<GestureArenaMember>>();
 
+  static final GestureArena instance = new GestureArena();
```

Comment:
Yeah, we can make the binding own it.  I was trying to make things less verbose, but statics are sadface.